tests/loggers/test_all.py

```diff
@@ -316,3 +316,67 @@ def _test_logger_created_on_rank_zero_only(tmpdir, logger_class):
     )
     result = trainer.fit(model)
     assert result == 1
+
+
+@pytest.mark.parametrize('logger_class', [
+    CometLogger,
+    MLFlowLogger,
+    NeptuneLogger,
+    TensorBoardLogger,
+    TestTubeLogger,
+    WandbLogger,
+])
+def test_logger_with_prefix_all(tmpdir, monkeypatch, logger_class):
+    _patch_comet_atexit(monkeypatch)
+    try:
+        _test_loggers_with_prefix(tmpdir, logger_class)
+    except (ImportError, ModuleNotFoundError):
+        pytest.xfail(f"pickle test requires {logger_class.__class__} dependencies to be installed.")
+
```

Comment:
@rohitgr7 Pretty sure this is not the correct way to handle this test. 
See the other tests, we need to mock the experiment dependency inside the logger, since we can't install it in CI.

Reply:
will check this.

Reply:
@awaelchli can't get this working. Can you check, please??